src/views/Welcome/index.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { useNavigation } from '@react-navigation/native'
+import React from 'react'
+import { View, Text, Image, TouchableOpacity } from 'react-native'
+import * as Animatable from 'react-native-animatable'
+import { styles } from './styles'
+
+export default function Welcome() {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏this screen is new?